### PR TITLE
9054 added summary report support for all qtypes

### PR DESCRIPTION
**Pull request by scttnlsn**

### CODE REVIEW

**Comment by scttnlsn** [comment]
Should I include similar specs for all question types?

File: app/models/report/summary_collection_builder.rb [hunk 1]
```diff
@@ -3,16 +3,24 @@ class Report::SummaryCollectionBuilder
...
+    time: "stat",
+    datetime: "stat",
+    select_one: "select",
+    select_multiple: "select",
+    date: "date",
+    text: "raw",
+    long_text: "raw",
+    barcode: "raw",
+    location: "raw",
+    image: "raw",
```
Comment (smoyth) [inline app/models/report/summary_collection_builder.rb]: What will image/video/audio summaries look like? Seems this would need more work. Thoughts?

Comment (scttnlsn) [reply app/models/report/summary_collection_builder.rb]: @smoyth It looks like it's not even possible to report on image/video/audio columns (those qtypes are not available in the report builder popup).

Comment (scttnlsn) [reply app/models/report/summary_collection_builder.rb]: Should I just remove them here?  Or do we want to include them in the report somehow?

**Pull request merged**